jupyterlab/commands.py

```diff
@@ -253,10 +308,70 @@ def uninstall_extension(name, app_dir=None):
 
 
 def list_extensions(app_dir=None):
-    """List installed extensions.
+    """List the extensions.
```

Comment:
Is there a way we can show the JupyterLab version in this information. That would help us in debugging the output of this as it relates to what the user sees in the UI. Also, we might want to somehow expose the app dir in the UI or JS console to help us debug these things. My concern is that users will have multiple builds and we will have a tough time figuring out which is which.